ambari-web/app/styles/theme/bootstrap-ambari.css

```diff
@@ -1143,6 +1143,8 @@ input.radio:checked + label:after {
   z-index: 100;
   top: 0;
   left: 50px;
+  max-height: calc(100% - 100px);
```

Comment:
Changes should be moved to ambari-web/app/styles/bootstrap_overrides.less

Reply:
Completed